client-http/src/test/scala/com/cloudera/livy/client/http/HttpClientSpec.scala

```diff
@@ -18,11 +18,15 @@
 
 package com.cloudera.livy.client.http
 
+import java.io.{ByteArrayInputStream, File}
 import java.net.{InetAddress, URI}
+import java.nio.file.{Paths, Files}
```

Comment:
hmm, this should have failed the style checker... in fact I don't see the style checker running in your test build, so I wonder what's going on.